app/assets/javascripts/patients.coffee

```diff
@@ -22,6 +21,39 @@ calculateRemainder = ->
 valueToNumber = (val) ->
   +val || 0
 
+getClinics = ->
```

Comment:
We have a clinics coffeescript file, no? Let's move stuff pertaining to this feature over there, since as you very correctly pointed out, the patients JS is getting a bit massive.